arbitrum-docs/how-arbitrum-works/07-interactive-fraud-proofs.mdx

```diff
@@ -0,0 +1,499 @@
+---
+title: Interactive Fraud Proofs
```

Comment:
We should add the word "Challenge" here. Maybe "Challenges: interactive fraud proofs" or something like that.